R/slicer.R

```diff
@@ -11,6 +11,38 @@ getTableOfSubtables <- function(dataTable, categoryName) {
   data.table(category = categories, dataTable = dataTables)
 }
 
+whichUiToRender <- function(ui, subtables, ns) {
+  switch(ui,
+         "tab" = {
+           tabPanelWithSlicerContent <- function(category, dataTable) {
+             tabPanel(category,
+                      slicerUI(ns(category)))
+           }
+
+           tabPanels <- pmap(subtables, function(category, dataTable) {
+             tabPanelWithSlicerContent(category, dataTable)
+           })
+
+           mainPanel(
+             do.call(tabsetPanel, tabPanels)
+           )
+         },
+         "box" = {
+           boxWithSlicerContent <- function(category, dataTable) {
+             shinydashboard::box(
+               width = 6, solidHeader = TRUE, collapsible = TRUE,
+               title = category, background = "light-blue",
```

Comment:
Light-blue background doesn't look good. Please revert to white.

Reply:
👍